content/sklearn/concepts/kernel-ridge-regression/kernel-ridge-regression.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'Kernel Ridge Regression'
+Description: 'Kernel ridge regression is a sophisticated linear regression model combined with L2 regularization and kernel trick to provide optimal solutions.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'Machine Learning'
+  - 'Python'
+  - 'Scikit-learn'
+CatalogContent:
+  - 'paths/intermediate-machine-learning-skill-path'
```

Comment:
``` suggestion
  - 'paths/intermediate-machine-learning-skill-path'
```
The course/path mentioned above is not valid. Using the course/path landing page slugs is necessary, If the path you want is not present in the [catalog-content. md file](documentation/catalog-content.md). You can add one.